### PR TITLE
Foodmeetings responsive

### PR DESCRIPTION
**Pull request by MirkoTV**
Making foodMeetings view "more responsive".

### CODE REVIEW

File: src/main/webapp/WEB-INF/foodmeeting/foodMeeting.jsp [hunk 1]
```diff
@@ -28,16 +28,16 @@
 				<h5 class="lighten-1 grey-text">We are loading your meetings right now!</h5>
 			</div>
 		</div>
-		<div class="hide">
-			<div id="new-meeting-placeholder" class="grid-item row" style="width:1000px" data-date="0">
+		<div class="hide">
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/foodMeeting.jsp]: remove the end tabs,

File: src/main/webapp/WEB-INF/foodmeeting/foodMeeting.jsp [hunk 2]
```diff
@@ -28,16 +28,16 @@
...
+			<div id="new-meeting-placeholder" class="grid-item row col s12" data-date="0">
 				<div id="new-meeting-hello-meessage" class="card col s12 m6 offset-m3 pointer center">
 					<div class="card-content">
 						<span class="grey-text pointer">You look hungry!... let's create a meeting!</span>
 					</div>
 				</div>
-			</div>
-			<div id="new-meeting" class='grid-item' style="width:1000px" data-date="0">
-				<div class="card row col s8 offset-s2">
+			</div>
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/foodmeeting/foodMeeting.jsp]: remove the end tabs,

File: src/main/webapp/WEB-INF/tags/template.tag [hunk 3]
```diff
@@ -44,6 +44,7 @@
     <script src="/assets/js/lib/jsrender.js"></script>
     <script src="/assets/js/lib/masonry.min.js"></script>
     <script src="/assets/js/lib/isotope.pkgd.min.js"></script>
+	<script src="/assets/js/lib/packery-mode.pkgd.min.js"></script>
```
Comment (rayaviri) [inline src/main/webapp/WEB-INF/tags/template.tag]: indentation

File: src/main/webapp/assets/js/foodMeetings.js [hunk 4]
```diff
@@ -35,47 +36,64 @@ $(function () {
...
 		var $newFoodMeeting = $($foodMeetingTmpl.render(data));
-		$newFoodMeeting.imagesLoaded().always(function(){
-			foodMeetings.isotope('insert', $newFoodMeeting);
+		$newFoodMeeting.imagesLoaded().always(function(){			
+			if(isNewMeetingFirst) {
+				$(".is-first").addClass('s6 l3');
+				$(".is-first").find('img').css('height', firstImageHeight + 'px');
+				$(".is-first").removeClass('is-first s12 l9');				
+			}
+
```
Comment (rayaviri) [inline src/main/webapp/assets/js/foodMeetings.js]: remove the tabs

File: src/main/webapp/assets/js/foodMeetings.js [hunk 5]
```diff
@@ -97,19 +115,18 @@ var FoodMeetingsList = function(foodMeetings, newMeetingPlaceholder){
...
-			sortAscending: true,
+			layoutMode: 'packery',
 			getSortData: {
 				date: function(itemElem) {
 					return $(itemElem).data("date");
 				}
 			},
 			sortBy : 'date',
-		});
+		});
```
Comment (rayaviri) [inline src/main/webapp/assets/js/foodMeetings.js]: remove tabs

File: src/main/webapp/WEB-INF/foodmeeting/foodMeeting.jsp [hunk 6]
```diff
@@ -28,16 +28,16 @@
...
 					</div>
 				</div>
-			</div>
-			<div id="new-meeting" class='grid-item' style="width:1000px" data-date="0">
-				<div class="card row col s8 offset-s2">
+			</div>			
+			<div id="new-meeting" class='grid-item col s12' data-date="0">
+				<div class="card row col s12">
 					<div class="new-meeting-img col s3 card-image waves-effect waves-block waves-light">
 						<img id="new-image-card-id" class="new-meeting-img" data-target="modal1"
```
Comment (rodrigouz) [inline src/main/webapp/WEB-INF/foodmeeting/foodMeeting.jsp]: improve data-target value name

**Review by rayaviri — COMMENTED** [review]
looks fine

**Review by rayaviri — APPROVED** [review]
looks fine

**Pull request merged**